eva_cttv_pipeline/evidence_string_generation/clinvar_to_evidence_strings.py

```diff
@@ -232,9 +232,9 @@ def generate_evidence_string(clinvar_record, allele_origins, disease_name, disea
         'variantRsId': clinvar_record.measure.rs_id,
 
         # PHENOTYPE ATTRIBUTES.
-        # The alphabetical list of *all* disease names from that ClinVar record.
-        'cohortPhenotypes': sorted([trait.preferred_or_other_valid_name for trait in clinvar_record.traits
-                                    if trait.preferred_or_other_valid_name is not None]),
+        # The alphabetical list of *all* valid disease names from all traits from that ClinVar record, reported as a
+        # flat list. See https://github.com/EBIvariation/eva-opentargets/issues/221 for a discussion of this choice.
+        'cohortPhenotypes': sorted([name for trait in clinvar_record.traits_with_valid_names for name in trait.all_valid_names]),
```

Comment:
In case the names are repeated for some reason, we don't want the duplicates:

```suggestion
        'cohortPhenotypes': sorted({name for trait in clinvar_record.traits_with_valid_names for name in trait.all_valid_names}),
```